packages/asc-ui/src/components/Radio/Radio.tsx

```diff
@@ -1,27 +1,85 @@
-import * as React from 'react'
-import RadioItemStyle from './RadioItemStyle'
-import RadioLabelStyle from './RadioLabelStyle'
-import RadioGroupStyle from './RadioGroupStyle'
+import React, { useState, useContext, useMemo } from 'react'
+import RadioStyle, {
+  RadioWrapperStyle,
+  RadioCircleStyle,
+  Props,
+} from './RadioStyle'
+import RadioContext from './RadioContext'
+import LabelContext from '../Label/LabelContext'
 
-interface Props {
-  id: string
-  name?: string
-  label: string
-  value?: string
-  defaultChecked?: boolean
-  disabled?: boolean
-}
+const Radio: React.FC<Props & React.HTMLAttributes<HTMLDivElement>> = ({
+  className,
+  onChange,
+  variant,
+  disabled,
+  id,
+  defaultChecked,
+  error: errorProp,
+  ...otherProps
```

Comment:
Is `value` one of the `otherProps`? If so, wouldn't a test that verifies that a radio button has the correct value be in order?

Reply:
Good point.`value` is one of the `otherProps` indeed.

Reply:
Added `value` to props and rewritten the test